addons/im_livechat/models/chatbot_script_step.py

```diff
@@ -367,6 +368,25 @@ def _process_step_forward_operator(self, discuss_channel):
 
         return posted_message
 
+    def _to_store(self, store: Store, /, *, fields=None):
+        if fields is None:
+            fields = {
+                "answers": None,
+                "message": [],
+                "type": True,
+            }
```

Comment:
Do you actually need an array/subkeys? Otherwise a list is the preferred format.
In other words, is there ever a non-default value for `answers`? I don't think it's used.

Later, sub-keys config will probably be passed from a "magic" item in the list (non-string), rather than a dict format, but I still have to implement that part.